### PR TITLE
Fix broken example

### PR DESCRIPTION
**Pull request by Jack-Dane**

### CODE REVIEW

**Comment by Jack-Dane** [comment]
Fixes an issue with the example raised in #1066.

**Comment by dieser-niko** [comment]
I would go a different way. The reason this fails at all is because the next url is `/playlists/<playlist_id>/tracks` while the initial request url is `/playlists/<playlist_id>`. It should be more straight forward with `sp.playlist_tracks` instead of `sp.playlist` so we can get a consistent response.

File: examples/user_playlists_contents.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def show_tracks(results):
             print(playlist['name'])
             print('  total tracks', playlist['tracks']['total'])
 
-            results = sp.playlist(playlist['id'], fields="tracks,next")
+            results = sp.playlist(playlist['id'], fields="tracks,next,items")
             tracks = results['tracks']
             show_tracks(tracks)
```
Comment (dieser-niko) [inline examples/user_playlists_contents.py:30]: Related to: https://github.com/spotipy-dev/spotipy/pull/1068#issuecomment-2089318419
```suggestion
            tracks = sp.playlist_tracks(playlist['id'], fields="items,next")
            show_tracks(tracks)
```

**Comment by Jack-Dane** [comment]
@dieser-niko, thanks for the review, I found the same when investigating the bug, but I didn't know the method you mentioned was available. 

I agree with your proposed changes, the slight change being that `playlist_tracks` is deprecated with the suggestion to use `playlist_items`. Still, it calls the same endpoint mentioned.

**Comment by dieser-niko** [comment]
I guess I missed the deprecation warning, there were just too many prints :P

One thing tho, even though the warning says to use `additional_types=('track',))`, it didn't say to remove the fields parameter. I think it would be better to add it since it was a part of the example before. In this case it should be `fields="items,next"`.

**Comment by Jack-Dane** [comment]
@dieser-niko, yep, my bad, updated to still include the `fields` kwarg.

**Review by dieser-niko — APPROVED** [review]

**Comment by Msoyosa** [comment]
Hi Jack, thank you for your plugin for CF7 integration with Odoo. Can I ask how to include the company name and marketing tag, the marketing tag has a Odoo field name  "x_studio_marketing_tag" - many to many, because they are not reflecting in Odoo, both the company name and marketing tag. Thank you.

**Comment by dieser-niko** [comment]
@Msoyosa please visit Jack Danes repository and open an issue there. The topic of your comment does not seem to be related to the Spotipy library and should not be discussed here.

**Pull request merged**